debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/TestHelper.java

```diff
@@ -140,7 +140,7 @@ public static PostgresConnectorConfig.LogicalDecoder decoderPlugin() {
      * @return the PostgresConnection instance; never null
      */
     public static PostgresConnection create() {
-        return new PostgresConnection(defaultJdbcConfig(), CONNECTION_TEST);
+        return new PostgresConnection(defaultJdbcConfig(), CONNECTION_TEST, "test-connector", "test-connector-thread", "test-task-1");
```

Comment:
nit: Apply at other places too.
```suggestion
        return new PostgresConnection(defaultJdbcConfig(), CONNECTION_TEST, "test-connector", "test-connector-thread", "0");
```